src/main/java/sirius/web/http/WebContext.java

```diff
@@ -753,7 +753,9 @@ private void installSessionPinningCookieIfRequired() {
         // session pin.
         givenSessionPin = Hashing.md5()
                                  .hashString(getRemoteIP().toString()
-                                             + TimeUnit.MILLISECONDS.toDays(System.currentTimeMillis()), Charsets.UTF_8)
+                                             + getHeader(HttpHeaderNames.USER_AGENT)
```

Comment:
getRemoteIp() already does that

Reply:
But since getRemoteIp only does that for Proxys known to the system and even then only takes the last IP from the X-FORWARDED-FOR list, I thought that maybe using the header directly would differantiate more in some cases.

I thought that its somehow likely that users with the same IP might have the same UA String, but looking further into it, this is only a problem if they do their FIRST login within the same minute, right?
Also, it might not be a big issue at all if these users have the same PIN, since they still use different session ?